python/cudf/cudf/tests/test_string.py

```diff
@@ -732,16 +674,20 @@ def test_string_index_duplicate_str_cat(data, others, sep, na_rep, name):
     )
 
 
-@pytest.mark.xfail(raises=ValueError)
-@pytest.mark.parametrize("sep", [None, "", " ", "|", ",", "|||"])
-@pytest.mark.parametrize("na_rep", [None, "", "null", "a"])
-def test_string_cat_str(ps_gs, sep, na_rep):
-    ps, gs = ps_gs
-
-    got = gs.str.cat(gs.str, sep=sep, na_rep=na_rep)
-    expect = ps.str.cat(ps.str, sep=sep, na_rep=na_rep)
-
-    assert_eq(expect, got)
```

Comment:
version?

Reply:
Did not version this as pandas still support `StringMethods` input but result in super ambiguous behaviour and hence they decided to deprecate now and remove it in the next release.